.gitignore

```diff
@@ -97,6 +97,7 @@ cover
 .venv/
 venv/
 *.json
+!/.devcontainer/devcontainer.json
```

Comment:
```suggestion
!.devcontainer/devcontainer.json
```

Reply:
I actually wanted to be explicit, hence the "full" path specification here